web/src/common/PoliciyTable.js

```diff
@@ -136,7 +136,7 @@ class PolicyTable extends React.Component {
         this.updateTable(table);
         Setting.showMessage("success", i18next.t("adapter:Delete policy successfully"));
```

Comment:
- "general:Successfully added"
- "general:Successfully deleted"
- "general:Successfully saved"
- "general:Failed to add: " + xxx
- "general:Failed to delete: " + xxx
- "general:Failed to save: " + xxx
- "general:Failed to connect to server"